proprietary/design-tokens/component/blockquote.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license SEE LICENSE.md
+ * Copyright (c) 2021 Gemeente Utrecht
+ * All rights reserved
+ */
+
+/**
+ * @tokens Components / Blockquote
+ */
+:root {
+  /* @presenter Sizing */
+  --utrecht-blockquote-margin-inline-start: 1.6em;
+  --utrecht-blockquote-margin-inline-end: 1.6em;
+  --utrecht-blockquote-margin-block: 1.6em;
```

Comment:
whoopsy I think these duplicate `--utrecht-blockquote-margin-block` should be ending with `-start` and `-end` ?

Reply:
also, I'm not sure if `margin` is what we're using here? As I think maybe components should not define the whitespace surrounding them?